front/src/components/Steps/StepperCompleted.tsx

```diff
@@ -40,7 +38,7 @@ const AccountCreated: React.FC<IAccountCreatedProps> = ({ userType }) => {
   return (
     <div className="flex justify-center flex-col items-center mt-10">
       <Text size="lg" align="center">
-        Votre compte est en cours de création
+        {label}
       </Text>
```

Comment:
I think that using the translation directly in each component would be better than passing it as a prop